### PR TITLE
Do 2726 auth refresh

### PR DESCRIPTION
**Pull request by dominiksteiner**
@joncodo there is currently a PR in tech review for hubCore to add the route to refresh the auth token

https://git.broadsoft.com/BroadSoftLabs/hubCore/pull/75

simulate it with a count on the sample app.

### CODE REVIEW

**Comment by dominiksteiner** [comment]
@joncodo the PR https://git.broadsoft.com/BroadSoftLabs/hubCore/pull/75 has been merged now on our end, hopefully will end up on staging during the next days

**Comment by joncodo** [comment]
Once it gets to prod, we can merge this. Others are working on it now

**Comment by joncodo** [comment]
@dominiksteiner feel free to merge this once your stuff is on prod

**Review by joncodo — APPROVED** [review]

**Pull request merged**
